cpp/src/parquet/column_reader.cc

```diff
@@ -884,19 +884,20 @@ class ColumnReaderImplBase {
     } else {
       switch (encoding) {
         case Encoding::PLAIN: {
```

Comment:
Can we simplify these different `case` statements? Unless I'm missing something, they're executing the same code. So this could be:
```cpp
      switch (encoding) {
        case Encoding::PLAIN:
        case Encoding::BYTE_STREAM_SPLIT:
        case Encoding::RLE:
        case Encoding::DELTA_BINARY_PACKED: 
        case Encoding::DELTA_BYTE_ARRAY:
        case Encoding::DELTA_LENGTH_BYTE_ARRAY:
          auto decoder = MakeTypedDecoder<DType>(encoding, descr_, pool_);
          current_decoder_ = decoder.get();
          decoders_[static_cast<int>(encoding)] = std::move(decoder);
          break;

        case Encoding::RLE_DICTIONARY:
          throw ParquetException("Dictionary page must be before data page.");

        default:
          throw ParquetException("Unknown encoding type.");
      }
```

Reply:
I agree changed.